Makefile

```diff
@@ -148,13 +148,18 @@ chrome: wget
 		sudo dpkg --install $${ARCHIVE} 2>/dev/null; \
 	fi
 
-# https://sites.google.com/a/chromium.org/chromedriver/downloads/version-selection
-chromedriver: wget unzip chrome
+# Pull ChromeDriver from Chrome For Testing
+# Need to create the CHROMEDRIVER_PATH and then move the files in because the
+# directory structure in chromedriver_linux64.zip has changed.
+# https://stackoverflow.com/a/76729048
+chromedriver: wget unzip chrome jq
 	if [[ ! -f "$(CHROMEDRIVER_PATH)" ]]; then \
-		CHROME_VERSION=$$(google-chrome --version | grep -ioE "[0-9]+\.[0-9]+\.[0-9]+"); \
-		CHROMEDRIVER_VERSION=$$(curl https://chromedriver.storage.googleapis.com/LATEST_RELEASE_$${CHROME_VERSION}); \
-		wget -q https://chromedriver.storage.googleapis.com/$${CHROMEDRIVER_VERSION}/chromedriver_linux64.zip; \
-		sudo unzip chromedriver_linux64.zip -d $$(dirname $(CHROMEDRIVER_PATH)); \
+		CHROME_VERSION=$$(google-chrome --version | grep -ioE "[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+"); \
+		CHROMEDRIVER_URL=$$(curl -s https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json | jq -r ".versions[] | select(.version == \"$${CHROME_VERSION}\") | .downloads.chromedriver[] | select(.platform == \"linux64\") | .url"); \
```

Comment:
This code assumes that for any non-CfT Google Chrome binary, there is a corresponding ChromeDriver binary release. Unfortunately this is not _yet_ true in a few edge cases — e.g. A/B testing releases, where different Chrome versions are released per platform, could currently result in the latest CfT having a lower patch version than the non-CfT version.

To make this code 100% robust, you’d still apply [Version Selection](https://chromedriver.chromium.org/downloads/version-selection) like before, but now using the new CfT JSON endpoints. Concretely, revert back to `grep -ioE "[0-9]+\.[0-9]+\.[0-9]+")` to get just the `MAJOR.MINOR.BUILD` parts of the version number, and then run it through https://googlechromelabs.github.io/chrome-for-testing/latest-patch-versions-per-build-with-downloads.json to find the corresponding ChromeDriver version there.